web/src/app/layout.tsx

```diff
@@ -15,16 +16,19 @@ export const metadata: Metadata = {
 export default function RootLayout({children,}: Readonly<{children: React.ReactNode;}>) {
   return (
     <html lang="en">
-      <head>
-        <ColorSchemeScript defaultColorScheme="dark"/>
-      </head>
+		<head>
+			<ColorSchemeScript defaultColorScheme="dark"/>
+		</head>
 
-      <body className={inter.className}>
-        <MantineProvider defaultColorScheme="dark">
-          <Header />
-          {children}
-        </MantineProvider>
-      </body>
+      <body className={`${inter.className} flex flex-col min-h-screen`}  >
+			<MantineProvider defaultColorScheme="dark">
+				<Header/>
+					<main className="flex-grow">
```

Comment:
@jonathanplacatka here is where I changed main

Reply:
oh I see makes sense